### PR TITLE
DCG-3410

### PR DESCRIPTION
**Pull request by tbuckley54**
Updated to latest sbt-docker-compose version

removed health check logic in favour of built in healthcheck to docker compose (https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck)

### CODE REVIEW

File: src/main/scala/com/trueconnectivity/e2e/E2eTestPlugin.scala [hunk 1]
```diff
@@ -23,34 +23,32 @@ object E2eTestPlugin extends AutoPlugin {
     import DockerComposePlugin.autoImport._
 
     lazy val settings: Seq[Setting[_]] = Seq(
-      testCasesJar := (IntegrationTest / packageBin / artifactPath)
-        .value
-        .getAbsolutePath,
+      testCasesJar := (IntegrationTest / packageBin / artifactPath).value.getAbsolutePath,
       testCasesPackageTask := {
         HealthCheck.waitUntilHealthy(
```
Comment (dominikschmid) [inline src/main/scala/com/trueconnectivity/e2e/E2eTestPlugin.scala]: I wonder why we even need that extra task. Can we not do the healthcheck directly in the compose file?(https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck)

But maybe not worth the effort to change if it works as is.

**Review by dominikschmid — APPROVED** [review]

File: src/main/scala/com/trueconnectivity/e2e/E2eTestPlugin.scala (hunk 1 above)
Comment (tbuckley54) [reply src/main/scala/com/trueconnectivity/e2e/E2eTestPlugin.scala]: i dont think that actually stops the docker compose command from completing, its just used to set the state of a container (starting, healthy, unhealthy)

Comment (dominikschmid) [reply src/main/scala/com/trueconnectivity/e2e/E2eTestPlugin.scala]: Right, we could maybe check the status and wait directly inside sbt-docker-compose (with filtering the container ids by health as well). Then we wouldn't need to assign and read the ports to env vars at all. But just an idea.

File: build.sbt [hunk 2]
```diff
@@ -44,7 +44,7 @@ addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.9.0")
 
 addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.7.6")
 
-addSbtPlugin("com.trueconnectivity" % "sbt-docker-compose" % "1.0.37")
+addSbtPlugin("com.trueconnectivity" % "sbt-docker-compose" % "1.0.38-SNAPSHOT+2-DCG-3410")
```
Comment (teroxik) [inline build.sbt]: don't forget to update once its released

**Review by dominikschmid — APPROVED** [review]

File: build.sbt [hunk 3]
```diff
@@ -53,7 +53,4 @@ pomIncludeRepository := { _ =>
 }
 
 addCommandAlias("format", "; scalafmt ; test:scalafmt ; scalafmtSbt")
-
-libraryDependencies += "org.scalaj" %% "scalaj-http" % "2.4.2"
```
Comment (teroxik) [inline build.sbt:57]: 👍

**Review by teroxik — APPROVED** [review]

**Pull request merged**